### DISCUSSION TITLE
2023-11-03 - Practice makes perfect - Chewing the Fat 📈

### DISCUSSION THREAD

**Discussion by GordonBeeming** [SSWConsulting/SSW.Rules.Content#7161] [Chewing the Fat]
Hey Presenters,

We would like some feedback on "Speaker Coach" in PowerPoint

[Do you know practice makes perfect?](https://www.ssw.com.au/rules/practice-makes-perfect/)

**Comment by ricksu978** [comment]
Value from "Speaker Coach"

1. Never notice my pitch was too low
2. It's good to see my pace was just about right!

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/35659961/75f0b49c-837a-4b4c-b841-0ebfc0320bec)

**Comment by tiagov8** [comment]
<img width="793" alt="Screenshot 2023-11-02 at 1 53 25 PM" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/12601228/59ad5b59-0151-42ad-b4f2-681a1715f9cf">

<img width="781" alt="Screenshot 2023-11-02 at 1 53 51 PM" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/12601228/fde4c34c-f597-4dcb-9563-c5b6dde5238a">

**Comment by brydeno** [comment]
It did make a few points. I did find that it's feedback made me worse for a short while afterwards. I will give this a try in more detail over full length stuff.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/77371877/b215da76-a1c3-4e91-addd-f4cdf1416138)

**Comment by GordonBeeming** [comment]
Interestingly, results and some learnings for other presentations from this demo. I'll be using this going forward 

- ⭐ I didn't know about the Pronunciation feature (it lets your practice them inline afterward too)
- From the real-time feedback, I realize I start my bullet topics with an "umm", almost like an exhale 😅

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5680199/dfc74126-61d0-416c-b187-43a7011b4abc)

Note: Results image is altered by moving the bottom boxes to the right for a wider image instead of a long page

**Comment by matt-goldman** [comment]
<img width="596" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/19944129/b3b090bb-92da-4ec9-9fc6-1f9cbfc83b89">
<img width="597" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/19944129/827a0f7c-4c66-4157-8caa-dcc9ee89599b">
<img width="598" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/19944129/901bac8c-23cd-44da-9a0f-b32ecc402186">

Calling BS on "like". I didn't use it as a filler word, I used it as a comparison word.

**Comment by jakebayliss** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/32888060/61f90533-89b3-45bf-9319-0be13676c624)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/32888060/ba735408-151d-436c-8cb7-9c049fbcb8a0)
Looks like im very monotone 🤖💀

**Comment by ozdemir-mehmet** [comment]
<img width="569" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/37358029/51223557-b5de-4bcb-8305-d12a1297b06d">
<img width="513" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/37358029/33b9e1c4-8662-4413-9d36-9cca7306dbf9">
<img width="467" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/37358029/d528fe2f-f74a-4457-97bb-9e3f8ec9c33c">

**Comment by Ben0189** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/87377903/cfb01b9a-c4d5-4c5f-9904-4a9ca300cdb1)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/87377903/d70c0207-2557-4131-9be5-c6c8029883c8)

Interesting tools🤔

**Comment by amankumarrr** [comment]
An interesting feature for rehearsing presentations. 

<img width="600" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/71385247/bd06d8f3-5449-4ab6-a99f-76d6a20da6aa">
<img width="600" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/71385247/ef656d89-048e-4572-8c30-4c55c359813a">
<img width="602" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/71385247/e49f7144-8290-413f-92e2-657aeff36984">

**Comment by AshAnil** [comment]
Interesting feature 

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/100336252/730ec1a5-afb6-4fc3-bd2e-5ed086145322)

**Comment by joshbermanssw** [comment]
<img width="793" alt="Screen Shot 2023-11-03 at 9 17 40 am" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/137844305/b43edaa2-f396-44eb-be66-e9b492406e59">
Definitely an interesting tool to play around with prior to presentations.

**Comment by bahjatalaadelssw** [comment]
![Screenshot 2023-11-03 at 9 20 44 am](https://github.com/SSWConsulting/SSW.Rules.Content/assets/108774015/bc4163a5-bf69-40c8-a812-9e01fba64ea3)
![Screenshot 2023-11-03 at 9 21 06 am](https://github.com/SSWConsulting/SSW.Rules.Content/assets/108774015/d77d0a11-8dc2-45e0-ba9e-b4cb7edd36f4)
![Screenshot 2023-11-03 at 9 21 15 am](https://github.com/SSWConsulting/SSW.Rules.Content/assets/108774015/42a8ca44-0445-4301-8210-b79574078e48)

**Comment by SamBWagner** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/66365977/a052a8f6-2ac9-4436-aa96-88d72a774033)

It was cool. But I'm a presentation god so no need for any assistance

**Comment by cvionnet** [comment]
This tool is really interesting!

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/46954368/c26d2514-ebaa-457b-8ea8-97782cdecab6)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/46954368/53f4bfd5-9ff1-402f-8c21-96666267ebb9)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/46954368/96d4f150-014d-408c-bb13-910b657c3b9a)

**Comment by wicksipedia** [comment]
looks like we need a ctf on how to take scrolling screenshots

![CleanShot 2023-11-03 at 09 28 13@2x](https://github.com/SSWConsulting/SSW.Rules.Content/assets/600044/0bbd7790-2f6b-46e1-8c78-fd665287daab)

**Comment by piers-sinclair** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/79821522/9cc9d54d-1012-439a-a1da-18823008e24a)

The tool doesn't seem that useful, but it's worth knowing about.

**Comment by 2fernandez** [comment]
![Screenshot 2023-11-03 at 9 33 53 am](https://github.com/SSWConsulting/SSW.Rules.Content/assets/27670973/1d5a02d7-6cc6-4e74-ac26-ec624a0857d3)

**Comment by MarcusIrmscherSSW** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/129343825/f1db4c07-aff6-48a5-b7af-cd72f1f08e90)

**Comment by tkapa** [comment]
It noticed that I was just reading from the slides. To be expected. I found it a useful tool while practicing for my User Group. 

<img width="791" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/17246482/483b28aa-b60e-4a8b-96c4-bf7dca04e127">

**Comment by tomek-i** [comment]
I see value in the tool to practice some of the slides
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/3291e5d7-4f2f-4deb-a826-27418b62c458)

**Comment by divgul** [comment]
The tool is very cool. Here were my results. 

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/85266364/979c9db3-0d5b-4169-9dc4-41e4d20c528c)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/85266364/131dde05-991b-4578-a023-68a474ef24db)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/85266364/3a1b1f8b-1b76-443a-904c-8a93dc2a4052)

**Comment by jeoffreyfischer** [comment]
![ppt-coach](https://github.com/SSWConsulting/SSW.Rules.Content/assets/106663901/46e0bcac-7b26-4c9b-8d8b-54ee4fbdae45)

**Reply by jeoffreyfischer** [reply]
Great tool ! I wasn't even aware of some of my flaws

**Comment by bettybondoc** [comment]
It's good to have a "coach" built in with PowerPoint for that initial feedback and improvement before you get a colleague to do a Test Please. Never got the comment "low pitch" before as I tend to be very high-pitched (bordering annoying) when I get too nervous or excited.

![Image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33026270/2b110881-0b7f-4dc9-b988-f2d569d64154)
![Image 3-11-2023 at 9 27 am](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33026270/8fc31b0c-cc2e-4e2b-97c5-695ce39bde5a)

**Comment by zacharykeeping** [comment]
Seems like a handy tool!

<img width="726" alt="Screenshot 2023-11-03 at 9 53 12 am" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/11418832/e79dfea7-c8ee-4557-b948-25709c83006b">

**Comment by camillars** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/53064613/57424098-d81b-46df-b44e-73be5c3d72fe)

**Comment by Harry-Ross** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/20507092/d729f763-ae95-413a-a348-509d1fbd1fc4)

Seems really useful - it detected that I pronounced the word "community" wrong, which was likely a false positive.

**Comment by JeanThirion** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33501311/94ba4266-8b48-4fd1-a04b-aee4f8cff6f4)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/33501311/9df81af6-e00b-4365-b59d-aa2356feda0e)

During my presentation I:
- Did a lot of back and forth between slides pretending to be lost (and saying it aloud obviously)
- Cursed a little, but never used the "C" word (so no idea what this is about)
- Read one of the slides entirely with the most robotic voice possible
- Went crazy fast and super slow
- Went stupidly over-the-top with my explanations on why choose SSW 
- Tried swearing in French

Most of these obvious and easy-to-detect flaws were not captured (probably because the report gives an "average" ?)
The tool is a great idea and has great potential, but not there yet. Oh and it's only working in English - it didnt detect any of the French I said.

**Comment by apforsyth** [comment]
great tool, one thing i noticed was that it mistook an inbreath for a filler

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/53065747/319f9bc0-7383-4b5b-8f90-065ba6a0e2c1)

**Comment by Marxoz** [comment]
Good tool. Pretty sure I didn't ummm, might have been my accent 😂
<img width="1000" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/86134557/2c4d9e83-1b51-451f-9b99-ea1186ba0246">

**Comment by jaydenalchin** [comment]
Pretty interesting feature!

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/75593567/2aa22bee-00a3-4e41-8fb4-4ca098b22fa0)

**Comment by NadeekaJK** [comment]
Sometimes the word "and" is interpreted as an "umm". 
![Coach](https://github.com/SSWConsulting/SSW.Rules.Content/assets/94897150/ca2cbf61-2bff-4a55-82f7-81690141ddbe)

**Comment by ncn-ssw** [comment]
Interesting, but not a substitute for listening to a recording of yourself.

![full-results](https://github.com/SSWConsulting/SSW.Rules.Content/assets/132531112/f21290a8-585d-4813-b8ec-8effedbcf15f)
**Figure: My results - demonstrating my tendency to go monotone if I need to think too hard about extra material to mention**

**Comment by steven0x51** [comment]
Didn't know this feature, very good tool
![Snag_46b4b203](https://github.com/SSWConsulting/SSW.Rules.Content/assets/41563445/167f31a3-f77e-4155-a9ac-96fcea2eab2a)

**Comment by chrisschultzssw** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/94873295/991b6f28-5254-40b1-b893-cc5a8a4b5746)

**Comment by sethdaily** [comment]
I gave a bad presentation and still got good results (see image).
My guess - it's probably more helpful to listen to or watch a recording of yourself.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/130118701/adbe5e96-01fc-49fe-98c7-e417a7d78763)
Figure: Still good results from a sub

**Comment by ae-cher-ssw** [comment]
Value from "Speaker Coach" - an interesting tool! 

Receiving feedback from a human is preferred, but cool that you have feedback on pitch/tone/pronunciation

 
<img width="386" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/147477898/c988574c-a733-4259-94da-c15ea3fe903d">

**Comment by jimmy1992abc** [comment]
This is a hard task for me😢
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/112846253/b393c5b7-fbb4-416e-86f2-060b2cb9ac6a)

**Comment by PennyWalker** [comment]
This was really cool, I knew I said 'umm' too much when presenting this just confirmed it! I also didn't realize that the tone i used wasn't upbeat enough, would be fun to keep practicing and seeing the results over time! Would be cooler if you could track your progress over time with a history.
![My-Presso1](https://github.com/SSWConsulting/SSW.Rules.Content/assets/25000905/e9f41a9f-a9da-4ef5-a576-ebdb6ab6989f)

**Comment by stephanfakossw** [comment]
It a great little feature. Awesome for practice, but...it does not store past reports so you can monitor your progress.

@tiagov8 Could we send this to MS as a better software suggestion?

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/96279310/ff43e325-b260-4a19-b0c9-6f05b7d37a77)

**Comment by leehawkinsssw** [comment]
There are some handy hints provided by this speaker coaching tool. Just remember to focus on telling a compelling story, though, and not trying to get a perfect score from this tool!

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/106290015/744b231e-213d-4e46-8e4d-905bb5dabb6d)

**Comment by MattParkerDev** [comment]
Fun tool, the small window is annoying and a way to export reports would be nice.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/61717342/ab9a6a6a-3b03-4790-9074-d16cb9f50669)


![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/61717342/052c94e1-9a29-4c89-9212-fcf6345cdc17)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/61717342/ed652119-518d-4fc4-b6f5-d3a06460c912)

**Comment by jackreimers** [comment]
Useful tool but definitely not a replacement for human feedback.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/10693364/64433a54-831b-4882-b3ab-4e02fd35bf11)

**Comment by leahy268** [comment]
Useful tool.  I might make a point of doing some of this before my next video.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/58454534/363e6891-579e-469f-ba28-b7330c11acbd)

**Comment by jernejk** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/ea5ebe97-97a4-4c37-bc34-2b0244d92a1d)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/5943653/a5db4269-58b3-4a8b-9972-d1fa45eb0258)

For this one, it didn't give me too much useful telemetry but I have used it in the past with useful information.
I failed first try because the real-time suggestion of "You're doing great" broke my flow and couldn't recover.

**Comment by Dhruv-0987** [comment]
<img width="802" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/69164713/4018ddbb-e9c9-450f-8523-1bab3dc0fe77">

The report sure is helpful but the popup while I am rehearsing is super annoying distracted my several time.

**Comment by tombui99** [comment]
This is such a cool feature, would definitely use it when I need to present in the future

<img width="748" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/67776356/2cf52811-0210-4031-8857-d296a7201448">

<img width="641" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/67776356/17505429-e180-4a34-afa5-749ebe70bc70">

<img width="334" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/67776356/f5376538-e771-4b58-8200-73715ce2784d">

**Comment by JackDevAU** [comment]
<img width="635" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/149561149/111c8ddf-f779-4594-9490-302d84d10f8f">

Not a bad tool to help test a presentation!
Very exciting to see this improve and for more awesome AI features to slowly improve a lot of tasks like presenting and public speaking!

**Comment by Calinator444** [comment]
I found that it's a little inconsistent. I intentionally used non-inclusive language ("guys" and "fellas") but it didn't pick up on it. It's a very cool feature regardless though.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/65635198/b2772edc-a7da-43b0-b172-a079f981f506)

**Comment by babakamyljanovssw** [comment]
Cool feature to practice for presentation, didn't know about it at all.
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/138548013/3e0af27d-38fa-448a-a400-814012788a98)

**Comment by Berber-Mama** [comment]
Wow...It's cool and gives you useful suggestions, I hope this could support Chinese and other languages soon.
![Yang-ppt-rehearsal-report](https://github.com/SSWConsulting/SSW.Rules.Content/assets/59909869/27cc61ab-8738-46ed-8173-b9a855582c25)

**Comment by sylhuang** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/23565403/1b44686d-57ad-45ab-837b-7007a35d47f7)

I wonder if it captures chuckle... Not intentional but I definitely did a few times while pretending to do the presentation

**Comment by KristenHu** [comment]
![27d773d6b02efdd9a218d93400cfa17](https://github.com/SSWConsulting/SSW.Rules.Content/assets/52522913/41d3e8bf-b69d-4034-a33a-d49b10d83504)
![010348d0c82606a53be6c1a087063b6](https://github.com/SSWConsulting/SSW.Rules.Content/assets/52522913/73edf208-8f5c-4b20-8a70-717eed4dc1b6)
![8f04ba4039784da2a8e9d3aca39bb3e](https://github.com/SSWConsulting/SSW.Rules.Content/assets/52522913/718c1c3e-be2c-497f-adbb-277e3f8cb9d7)
It's a useful tool, I'll try it next time

**Comment by Aibono1225** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/127192800/064ecb82-7ec8-4d89-b233-0dd065285364)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/127192800/e75a90b5-efd1-4da2-bcee-630d8f9bd936)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/127192800/b5f0c58d-4a09-41a3-8864-63165ad3cc9f)

It gave me many good advices, very useful!

**Comment by jimmidier** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/16027480/50ae6be6-d7b2-4a72-a87a-91d3db3608c8)
Such a great tool, I'll definitely use it for my next presentation

**Comment by JerryLuo5799** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3375764/a0e48700-f980-481c-9f59-d420f0a37f7b)

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3375764/fcb0847e-6e41-493c-a2ca-961fa51ef1ec)

**Comment by AnkyChe** [comment]
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/59987641/58355b4c-63ec-4296-bb9c-a4504a6e9a83)

**Comment by jimmy1992abc** [comment]
My Chinese fellow use "Ummm" a lot, just like me🤣

**Comment by zl24622** [comment]
<img width="785" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/8521734/ab41bc5a-f3db-4ed2-ad25-c782496a7838">

<img width="789" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/8521734/85b820f2-56bf-41a0-892c-e059d9738d32">

**Comment by tino-liu** [comment]
Very interesting tool! 
<img width="801" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/18450582/99097967-f535-4619-a615-3babaebff9de">
<img width="796" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/18450582/d175d800-43e7-4895-b75e-1fa1f6dc7772">
<img width="801" alt="image" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/18450582/23e2d0af-95e9-425d-9435-fb59cb887ed7">

**Comment by PothieuG** [comment]
Very interesting tool. I will probabaly use it when I will have to prepare a presentation !
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/45673557/ff7693ed-c207-4470-93a3-1c2500e3c6f0)

**Comment by BrookJeynes** [comment]
<img width="787" alt="Screenshot 2023-11-06 at 07 39 32" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/25432120/d1c4eff3-f536-4864-8c75-05802a47e5bb">

<img width="785" alt="Screenshot 2023-11-06 at 07 39 41" src="https://github.com/SSWConsulting/SSW.Rules.Content/assets/25432120/9a53aff1-94c9-440a-8691-69b1e026e953">

**Comment by ozair-ashfaque** [comment]
Impressive tool, I would like to see the results when I have prepared for the presentation
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/96504411/d69e8bcd-de2e-45ce-9b4e-4e665fc7f938)

**Comment by vladislav-kir** [comment]
It was interesting to see that it noticed that my pitch variation was low.

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/54652701/2ddd6281-28c4-48a6-b415-1927f83c6f81)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/54652701/12322854-7524-4795-b681-c1c9071329f0)
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/54652701/77b7fa95-9638-4179-af9b-68872cad2e45)

**Comment by bradystroud** [comment]
I have been using the same feature in ClipChamp for done videos, it is cool.

- It didn't like my Australian pronunciation of 'certified' (no 'r'). I wanted to test if changing the language to en au would have the same problem but I couldn't reproduce it on en us
- 💡Would be cool if it could play back your mistakes (I'm pretty sure I didn't say ummmmm and ahhhhhh at all)

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/38869720/cc91c0a6-06dd-4bbd-9221-7bc317cfe90d)
**Figure: looking at this discussion, we should do a CTF on scrolling screenshots**

**Comment by Freego1783** [comment]
Did not know about this and I'll clearly use it when preparing a presentation ! 

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/4101360/9542b3d9-367d-47d4-934d-4eb6fdafda8f)

**Comment by UlyssesMaclaren** [comment]
Didn't work for me. 0 words per minute? I'm guessing my mic is not connecting:
![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/45545178/9100ec61-1d9a-4275-ae0d-7733cf1b6038)

**Comment by AlexRothwell** [comment]
The rehearsal coach looks like a fantastic tool, I can see it being used to know when you've rehearsed enough to know the presentation well enough to present for real

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/7465195/270f0594-d636-41b5-a916-eefa94b01de2)

My filler words here absolutely make sense as it was the first time I'd seen the slides so some more practice would help me become more familiar with the content. The same would go for pitch, once I know the points I want to hit, pitch variation is easier

I'm not convinced that I had the pronunciation of Linux wrong as the example is how I think I pronounce it so it'd be great it if had short snippets from your rehearsal to show you what it thought could be improved

**Comment by danielmackay** [comment]
I have seen this feature before, but had never tried it.  I thought this was quite valuable and provides feedback in many areas of the presentation.  Great tool to have in a presenters toolbelt!

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/2636640/eca20812-1e47-4db9-a2f6-125e797d3d08)

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/2636640/5a92c2ae-22b4-40b5-b4b0-36718888bd87)
